tasks/scripts/docker-semver-tags

```diff
@@ -3,26 +3,40 @@
 set -e -u
 
 version=$(cat version/version)
+if [ ! -z "$SUFFIX" ]; then
```

Comment:
oops, this has to move down...

Reply:
right, per my comment, how could a reviewer help in this case